webapp/javascript/services/exemplarsTestData.ts

```diff
@@ -0,0 +1,252 @@
+export const heatmapMockData = {
```

Comment:
I guess this file doesn't have to exist anymore?

Reply:
we can use it to mock `/api/exemplars` response